ui/src/pages/projects.js

```diff
@@ -31,9 +31,11 @@ const Projects = ({ location }) => {
     queryParams && queryParams.view ? queryParams.view : FILTERS.all,
   )
   const [isFilterOpen, setIsFilterOpen] = useState(false)
-  const [selectedOrderBy, setSelectedOrderBy] = useState(ORDER_BY['Name'])
+  const [selectedOrderBy, setSelectedOrderBy] = useState(
+    selectedOrderBy ? selectedOrderBy : ORDER_BY['Name']
```

Comment:
I would expect that `selectOrderBy || ORDER_BY['Name']` will work faster

Reply:
I chose to follow the existing coding style, which is important for overall readability.

A quick read on "short circuit evaluation" makes me doubt it has faster execution time (as it is in effect a conditional statement much like there is now) and it has a risk for unintended side-effects.

The PR should be open for adjustment by project owners, so they can opt to change if they feel it's better.